ONSdigital/dp-census-atlas
# env config files for prod/develop specific builds

### What

Add env config files for develop and prod environments that
override defaults when building for these envs. Currently only
the VITE_API_BASE_URL is added (which ensures prod deployments
talk to the prod geodata api and develop deployments to the develop
geodata api).

Refactor package.json, Makefile and build.sh scripts to source
appropriate env-specific .env file when building. Build.sh
looks at git history to determine env (develop branch = develop env,
master branch = prod env, anything else uses default local development
config)

### How to review

- check the preview works (netlify should use the 'normal' ec2 url)
- check npm run dev works locally (should also use the ec2 url)
- if you have develop access, run `npm run build-node-develop`, then `node build/index.js` and check the atlas works ok (should use the develop api url)

### Who can review

Anyone but especially @petem-methods
